packages/orchestration/src/transaction/transaction-orchestrator.ts

```diff
@@ -304,11 +374,13 @@ export class TransactionOrchestrator extends EventEmitter {
       const stepDef = flow.steps[step]
       const curState = stepDef.getStates()
       if (
-        curState.state === TransactionState.DONE ||
+        [TransactionStepState.DONE, TransactionStepState.TIMEOUT].includes(
+          curState.state
+        ) ||
```

Comment:
question(non-blocking): Why are we changing the state to NOT_STARTED for these cases?

Reply:
Makes sense, thanks for explaining